src/Umbraco.Web.UI.Client/src/common/mocks/services/localization.mocks.js

```diff
@@ -84,7 +84,7 @@ angular.module('umbraco.mocks').
                   "buttons_save": "Save",
                   "buttons_saveAndPublish": "Save and publish",
                   "buttons_saveToPublish": "Save and send for approval",
-                  "buttons_showPage": "Preview",
+                  "buttons_showPage": "Save and preview",
```

Comment:
@Matthew-Wise maybe it would be better the add a new language key here `buttons_saveAndPreview` since we already have `buttons_saveAndPublish`, `buttons_saveToPublish` etc.

Reply:
@Matthew-Wise Can you pick this up? I think its worth making it consistent

Reply:
Done :)